### PR TITLE
feat(airgap): Support scp files to node's data-dir

### PR DESCRIPTION
**Pull request by orangedeng**
Related issue: https://github.com/cnrancher/autok3s/issues/578

### CODE REVIEW

File: pkg/airgap/file_scp_test.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	type testcase struct {
+		name       string
+		args       string
+		expectPath string
+	}
+	for _, c := range []testcase{
+		{name: "no extra args", expectPath: defaultDataDirPath},
+		{name: "no data path args", args: "--bind-address=0.0.0.0", expectPath: defaultDataDirPath},
+		{name: "data dir args", args: "--data-dir /data", expectPath: "/data"},
+		{name: "data dir args with show name", args: "-d=/data", expectPath: "/data"},
```
Comment (JacieChao) [inline pkg/airgap/file_scp_test.go:19]: Is it should be `data dir args with short name`?

File: pkg/airgap/file_scp.go [hunk 2]
```diff
@@ -21,12 +22,16 @@ import (
...
+	dataDirSubpath string
+	targetPath     string
 }
 
 const (
-	installScriptName = "install.sh"
+	installScriptName      = "install.sh"
+	defaultDataDirPath     = "/var/lib/rancher/k3s"
+	dataDirParamPrefix     = "--data-dir"
+	dataDirParamPrefixShot = "-d"
```
Comment (JacieChao) [inline pkg/airgap/file_scp.go:34]: dataDirParamPrefixShort

File: pkg/airgap/file_scp.go [hunk 3]
```diff
@@ -283,3 +302,33 @@ func getScpFileMap(arch string, pkg *common.Package) (map[string]fileMap, error)
...
+			prefix = dataDirParamPrefix
+		}
+		if strings.HasPrefix(arg, dataDirParamPrefixShot) {
+			prefix = dataDirParamPrefixShot
+		}
+		if prefix == "" {
+			continue
+		}
+		// if the arg == dataPath prefix, return the next arg
+		if len(arg) == len(prefix) && i < len(args)-1 {
```
Comment (JacieChao) [inline pkg/airgap/file_scp.go:321]: Is `--data-dir=/data` or `-d /data` also valid for this situation?

Comment (orangedeng) [reply pkg/airgap/file_scp.go:321]: It should be valid

Comment (orangedeng) [reply pkg/airgap/file_scp.go:321]: Added test cases

**Review by JacieChao — APPROVED** [review]
LGTM

**Pull request merged**